app/models/storage_location.rb

```diff
@@ -162,105 +80,91 @@ def self.import_csv(csv, organization)
     end
   end
 
+  # NOTE: We should generalize this elsewhere -- Importable concern?
   def self.import_inventory(filename, org, loc)
     current_org = Organization.find(org)
     adjustment = current_org.adjustments.create(storage_location_id: loc.to_i, comment: "Starting Inventory")
-    CSV.parse(filename, headers: false) do |row|
+    # NOTE: this was originally headers: false; it may create buggy behavior
+    CSV.parse(filename, headers: true) do |row|
       adjustment.line_items
                 .create(quantity: row[0].to_i, item_id: current_org.items.find_by(name: row[1]))
     end
-    adjustment.storage_location.intake!(adjustment)
+    adjustment.storage_location.increase_inventory(adjustment)
   end
 
-  # Used to move inventory between StorageLocations; reflects items being physically moved
-  # Ex: move 500 size "2" diapers from main warehouse to overflow warehouse because insufficient space in main warehouse
-  def move_inventory!(transfer)
-    updated_quantities = {}
-    item_validator = Errors::InsufficientAllotment.new("Transfer items exceeds \
-                                                        the available inventory")
-    transfer.line_items.each do |line_item|
-      inventory_item = inventory_items.find_by(item: line_item.item)
-      new_inventory_item = transfer.to.inventory_items.find_or_create_by(item: line_item.item)
-      next if inventory_item.nil? || inventory_item.quantity.zero?
+  # FIXME: After this is stable, revisit how we do logging
+  def increase_inventory(itemizable_array)
+    itemizable_array = itemizable_array.is_a?(Array) ? itemizable_array : itemizable_array.to_a
```

Comment:
I'm not sure if this will work - the `.to_a` method we designed is a custom one. I'd consider us removing this fully, but I think @armahillo likes this pattern.

Reply:
I think it _will_ work. I wrote this little script that verifies that both `Array()` and `to_a` will a) call the customized to_a method if the object is not already an array, and b) return the original object if it is an array:

```ruby
#!/usr/bin/env ruby

class A
  def to_a
    [1,2,3]
  end
end

a = A.new

p Array(a) # [1, 2, 3]
p a.to_a   # [1, 2, 3]

b = [4,5,6]
p Array(b) # [4, 5, 6]
p b.to_a   # [4, 5, 6]
```

Reply:
You are correct.  That's way better!